### PR TITLE
cnf network: Update SriovMetricsExporter Tests

### PR DESCRIPTION
**Pull request by ajaggapa**

### CODE REVIEW

**Review by evgenLevin — APPROVED** [review]
lgtm

File: tests/cnf/core/network/sriov/tests/metricsExporter.go [hunk 1]
```diff
@@ -407,6 +408,13 @@ func createTestResources(testRes testResource) *pod.Builder {
 	_, err = testRes.network.Create()
 	Expect(err).ToNot(HaveOccurred(), "Failed to create SR-IOV network")
 
+	By("Verify NAD is created to proceed for Pod creation")
```
Comment (gkopels) [inline tests/cnf/core/network/sriov/tests/metricsExporter.go:411]: "Verify NAD is created to proceed with Pod creation"

Comment (ajaggapa) [reply tests/cnf/core/network/sriov/tests/metricsExporter.go]: Updated 👍

**Comment by gkopels** [comment]
lgtm

**Review by gkopels — APPROVED** [review]

**Pull request merged**